### PR TITLE
PR: Align plugin tabs to the left and center dockwidget ones

### LINKED ISSUE #9763: Align plugin tabs to the left and center dockwidget tabs for all OSes
## Problem Description
Plugin tabs are not aligned to the left in Mac OS

Screen Shot 2019-07-05 at 7 27 06 PM

### PR DESCRIPTION
**Pull request by juanis2112**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Aligned tabs to the left in console, editor and preferences.
Screen Shot 2019-07-10 at 12 53 44 AM
Screen Shot 2019-07-10 at 12 53 54 AM

### Issue(s) Resolved

Fixes #9763 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:
Juanis2112

### CODE REVIEW

File: spyder/plugins/editor/confpage.py [hunk 1]
```diff
@@ -325,3 +325,5 @@ def enable_tabwidth_spin(index):
         vlayout = QVBoxLayout()
         vlayout.addWidget(tabs)
         self.setLayout(vlayout)
+        self.setStyleSheet("QTabWidget::tab-bar {"
```
Comment (goanpeca) [inline spyder/plugins/editor/confpage.py]: Let's define this style sheet in a centralized location so we can import related things from there.

We could use `spyder/config/gui.py` or create a new file either at

`spyder/config/style.py` or at `spyder/utils/style.py`

Suggestions @ccordoba12 ?

Comment (ccordoba12) [reply spyder/plugins/editor/confpage.py]: For now I think it's fine to apply it on a widget basis.

@goanpeca, once you refactor things, then I agree with moving this to `spyder/utils/style.py`.

Comment (ccordoba12) [reply spyder/plugins/editor/confpage.py]: @juanis2112, please try to set this stylesheet rule in the parent class of our config pages, which is called `PluginConfigPage` and it's placed in `spyder/api/preferences.py`.

That way you would avoid to apply it to every config page separately.

**Comment by ccordoba12** [comment]
@juanis2112, please merge with master to fix the merge conflict here.

File: spyder/plugins/editor/confpage.py (hunk 1 above)
Comment (ccordoba12) [reply spyder/plugins/editor/confpage.py]: I was wrong here, as @juanis2112 let me knew yesterday. However, we can apply the stylesheet rule in `show_preferences` of `MainWindow`.

**Comment by ccordoba12** [comment]
@juanis2112, please see my last commit. I finally found how to center dockwidget tabs!

**Pull request merged**